packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.html

```diff
@@ -0,0 +1,447 @@
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+<div ui-view>
+    <div class="row d-lg-flex mb-3">
+        <div class="col-md-6">
+            <oui-tile
+                data-heading="{{ ::'pci_projects_project_training_dashboard_information' | translate }}"
+                class="h-100"
+            >
+                <div
+                    class="oui-tile__item"
+                    data-ng-if="$ctrl.regions.length > 1"
+                >
+                    <dl class="oui-tile__definition">
+                        <dt class="oui-tile__term">
+                            <span
+                                data-translate="pci_projects_project_training_dashboard_region"
+                            ></span>
+                        </dt>
+
+                        <oui-tile-description>
+                            <oui-dropdown placement="end">
+                                <oui-dropdown-trigger
+                                    text="{{ ::$ctrl.currentRegion.name | translate }}"
+                                ></oui-dropdown-trigger>
+                                <oui-dropdown-content>
+                                    <oui-dropdown-item
+                                        data-ng-repeat="entry in $ctrl.regions track by $index"
+                                        data-on-click="$ctrl.changeRegion(entry)"
+                                        >{{ entry.name }}</oui-dropdown-item
+                                    >
+                                </oui-dropdown-content>
+                            </oui-dropdown>
+                        </oui-tile-description>
+                    </dl>
+                </div>
+                <div class="oui-tile__item">
+                    <dl class="oui-tile__definition">
+                        <dt class="oui-tile__term">
+                            <span
+                                class="ellipsis"
+                                data-translate="pci_projects_project_training_dashboard_console_url"
+                            ></span>
+                            <button
+                                type="button"
+                                class="oui-popover-button ellipsis"
+                                data-oui-popover="{{:: 'pci_projects_project_training_dashboard_console_url_help' | translate}}"
+                            ></button>
+                            <img
+                                data-ng-src="{{:: $ctrl.illustration }}"
+                                alt
+                                class="w-25 float-right"
+                            />
+                        </dt>
+
+                        <oui-tile-description>
+                            <a
```

Comment:
this link won't have any text for screen readers because of the aria-hidden icon,
please add some description :
```
<a>
  <span class="oui-icon ..." aria-hidden="true"></span>
  <span class="sr-only">description</span>
</a>
```

Reply:
fixed